src/main/java/ro/secur/auth/exceptions/ExceptionModel.java

```diff
@@ -0,0 +1,23 @@
+package ro.secur.auth.exceptions;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+import java.time.LocalDateTime;
+
+
+@Getter
+@Setter
+public class ExceptionModel {
+
+    private String message;
+    private HttpStatus httpStatus;
+    private LocalDateTime timestamp;
+
+    public ExceptionModel(String message, HttpStatus httpStatus, LocalDateTime timestamp) {
```

Comment:
Please remove the constructor and add @AllArgsConstructor from lombok.

Reply:
yup